.github/workflows/update_licenses.yml

```diff
@@ -0,0 +1,42 @@
+name: "Update licenses"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 1 * *" # run once a month
+
+env:
+  BRANCH_NAME: "license-autoupdate"
```

Comment:
Do we need this? It's just used once. We could insert the branch name there.

Reply:
leftover, removed